release_notes/Service_Fabric_ReleaseNotes_90CU2.md

```diff
@@ -0,0 +1,95 @@
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 9.0 Cumulative Update 2.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current Breaking Changes](#current-breaking-changes)
+* [Future Retirement and Deprecation Path Callouts](#future-retirement-and-deprecation-path-callouts)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Windows Server 2022 is now supported as of the 9.0 CU2 release.
+* Support for using Service Fabric with Windows Server 2022 and Mirantis Container Runtime. Please see [Containers on Windows](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=dockerce#windows-server-1) documentation
+
+## Current Breaking Changes
+
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
+    Following scenarios are impacted:<br/>
+    * An application with guest executables or containers is created or upgraded in a Service Fabric 7.1+ cluster.<br/>
+    The cluster is then downgraded to a previous Service Fabric runtime version (e.g. Service Fabric 7.0).<br/>
+    The application fails to activate.<br/>
+    * A cluster upgrade from a version prior to Service Fabric v7.1 to Service Fabric 7.1+ version is in progress.<br/>
+    In parallel with the Service Fabric runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+    The Service Fabric runtime upgrade starts rolling back (due to any reason) to a version prior to Service Fabric runtime v7.1.<br/>
+    The application fails to activate.<br/>
+    To avoid issues when upgrading from a version prior to Service Fabric runtime v7.1 to a Service Fabric 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the Service Fabric runtime upgrade is in progress.<br/>
+    * The simplest mitigation, when possible, is to delete and recreate the application in Service Fabric 7.0.<br/>
+    * The other option is to upgrade the application in Service Fabric 7.0 (for example, with a version only change).<br/>
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+<br>
+
+## Future Retirement and Deprecation Path Callouts
+* Service Fabric ASP.NET Core packages built against ASP.NET Core 1.0 binaries are out of support. Starting Service Fabric 8.2, we will be building Service Fabric ASP.NET Core packages against .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, .NET Framework 4.6.1. As a result, they can be used only to build services targeting .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, >=.NET Framework 4.6.1 respectively.
```

Comment:
should we remove .net 5 from this list given its out of support?

Reply:
same comment for others below

Reply:
For now I resolving this comment and will re edit it once there is a clarification around it